app/src/main/kotlin/com/toanns/kotlinexample/service/ChatHeadService.kt

```diff
@@ -0,0 +1,120 @@
+package com.toanns.kotlinexample.service
+
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.graphics.PixelFormat
+import android.os.IBinder
+import android.util.Log
+import android.view.Gravity
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.view.WindowManager
+import android.view.WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE
+import android.view.WindowManager.LayoutParams.TYPE_PHONE
+import android.widget.ImageView
+import com.toanns.kotlinexample.R
+
+
+
+
+/**
+ * Copyright © 2016 AsianTech inc.
+ * Created by toanns on 22/12/2016.
+ */
+
+class ChatHeadService : Service() {
+    private var windowManager: WindowManager? = null
+    private var chatHead: ImageView? = null
+
+
+//    override fun onBind(intent: Intent?): IBinder {
+//        throw UnsupportedOperationException("not implemented") //To change body of created functions use File | Settings | File Templates.
+//    }
+
+    override fun onBind(intent: Intent): IBinder? {
+        return null
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+
+        windowManager = getSystemService(Context.WINDOW_SERVICE) as WindowManager?
+        chatHead = ImageView(this)
+        chatHead!!.setImageResource(R.mipmap.ic_launcher)
+
+        Log.d("ttt", "hay hay" + windowManager + " " + chatHead)
+
+        val params: WindowManager.LayoutParams = WindowManager.LayoutParams(
+                WRAP_CONTENT,
+                WRAP_CONTENT,
+                TYPE_PHONE,
+                FLAG_NOT_FOCUSABLE,
+                PixelFormat.TRANSLUCENT
+        )
+        params.gravity = Gravity.TOP or Gravity.LEFT
+        params.x = 0
+        params.y = 100
+
+        windowManager!!.addView(chatHead, params)
+
+        /*chatHead!!.setOnTouchListener ({ view, motionEvent ->
+            var initialX: Int = 0
+            var initialY: Int = 0
+            var initialTouchX: Float = 0f
+            var initialTouchY: Float = 0f
+            when(motionEvent.actionMasked) {
+                MotionEvent.ACTION_DOWN -> {
+                    initialX = params.x
+                    initialY = params.y
+                    initialTouchX = motionEvent.rawX
+                    initialTouchY = motionEvent.rawY
+                    return true
+
+                }
+            }
+        })*/
+
+
+        chatHead!!.setOnTouchListener(object : View.OnTouchListener {
+            private var initialX: Int = 0
+            private var initialY: Int = 0
+            private var initialTouchX: Float = 0.toFloat()
+            private var initialTouchY: Float = 0.toFloat()
+
+            override fun onTouch(v: View, event: MotionEvent): Boolean {
```

Comment:
WARNING: [ClickableViewAccessibility] `com/toanns/kotlinexample/service/ChatHeadService$onCreate$1#onTouch` should call `View#performClick` when a click is detected
 If a `View` that overrides `onTouchEvent` or uses an `OnTouchListener` does not also implement `performClick` and call it when clicks are detected, the `View` may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in `View#performClick` as some accessibility services invoke `performClick` when a click action should occur.